Cilsil.Test/E2E/NPETest.cs

```diff
@@ -178,6 +178,60 @@ public void ResourceLeakStaticSingleton(bool staticSingleton, InferError expecte
                                GetString(expectedError));
         }
 
+                /// <summary>
+        /// Validates that a resource leak on a StreamReader initialized in exception handling block 
+        /// is identified.
+        /// </summary>
+        /// <param name="blockKind">The kind of exception handling block expected to wrap the resource.</param>
+        /// <param name="closeStream">If <c>true</c>, invokes the Close method; otherwise,
+        /// does not.</param>
+        /// <param name="expectedError">The kind of error expected to be reported by Infer.</param>
+        [DataRow(BlockKind.Using, false, InferError.None)]
+        [DataRow(BlockKind.TryCatchFinally, true, InferError.None)]
+        [DataRow(BlockKind.TryCatchFinally, false, InferError.DOTNET_RESOURCE_LEAK)]
+        [DataTestMethod]
+        public void ResourceLeakExceptionHandling(BlockKind blockKind,
+                                                  bool closeStream,
```

Comment:
hmmm would it be hard to move the resource leak tests to a separate file (ResourceLeakTest.cs)? Else, we should probably rename NPETest, as inclusion of non-NPE warnings makes "NPETest.cs" a misnomer.

Reply:
Done. Renamed to InferSharpTest.cs